### PR TITLE
fix(presets): Update @aws-lambda-powertools monorepo url

### PR DESCRIPTION
**Pull request by TheHolyWaffle**
## Changes

The aws typescript powertools monorepo has changed url:

Previously: https://github.com/awslabs/aws-lambda-powertools-typescript

Current: https://github.com/aws-powertools/powertools-lambda-typescript

## Context

I noticed that the grouping behaviour was broken for these powertools npm packages

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Comment by rarkins** [comment]
It's usually best to included both for some time

File: lib/config/presets/internal/monorepo.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ const repoGroups = {
   ],
   'aws-cdk': 'https://github.com/aws/aws-cdk',
   'aws-lambda-powertools-typescript':
-    'https://github.com/awslabs/aws-lambda-powertools-typescript',
+    'https://github.com/aws-powertools/powertools-lambda-typescript',
```
Comment (viceice) [inline lib/config/presets/internal/monorepo.ts:44]: convert to array and add both urls, so it's working for old and new packages

**Review by viceice — CHANGES_REQUESTED** [review]

File: lib/config/presets/internal/monorepo.ts (hunk 1 above)
Comment (TheHolyWaffle) [reply lib/config/presets/internal/monorepo.ts:45]: Done

**Comment by TheHolyWaffle** [comment]
Ready for re-review

**Review by rarkins — APPROVED** [review]

**Review by viceice — APPROVED** [review]

**Pull request merged**